docs/spec/rpc/swagger.yaml

```
@@ -0,0 +1,2680 @@
+swagger: "2.0"
+info:
+  version: "0.32.1"
+  title: RPC client for Tendermint
+  description: A REST interface for state queries, transaction generation and broadcasting.
+  license:
+    name: Apache 2.0
+    url: https://github.com/tendermint/tendermint/blob/master/LICENSE
+tags:
+  - name: Websocket
+    description: Subscribe/unsubscribe are reserved for websocket events.
+  - name: Info
+    description: Informations about the node APIs
+  - name: Tx
+    description: Transactions broadcast APIs
+  - name: ABCI
+    description: ABCI APIs
+  - name: Evidence
+    description: Evidence APIs
+schemes:
+  - https
+host: stargate.cosmos.network:26657
```

Should we be using this address here? Due to the more generic nature of Tendermint, I'd imagine we'd need to provide an address that's relevant to a broader user base. The default for Tendermint configuration at the moment is to bind to `127.0.0.1`, so should we perhaps not just use:
```suggestion
host: localhost:26657
```

---

I do believe this is so when a user comes to the swagger docs they can hit the endpoint and get results back instead of having to spin a local node to be able to test it. similar to the try it out button here: https://cosmos.network/rpc/#/ICS0/get_node_info

---

Perhaps we should look into setting up a Tendermint/kvstore node here instead then, running the latest version of Tendermint? Something generic where users can follow along in the Tendermint documentation to interact with the server.